lib/xclarity_client/error/connection_failed_unknown.rb

```diff
@@ -0,0 +1,10 @@
+module XClarityClient
+  module Error
+    class ConnectionFailedUnknown < StandardError
+      def initialize(msg)
+        msg = "Connection falied, #{msg}."
```

Comment:
Should we check _msg_ for _nil_ like you do in the other error classes?

Reply:
Yes, fixed this. Thanks for your review.